### ISSUE TITLE
An ADA issue: the keyboard focus is not visible for elements such as image links, check-box, combo box

### ISSUE THREAD

**Issue by dmitry-fedyuk** [kitchencollection-com/core#25]
[upwork.com/ab/f/contracts/20874295](https://www.upwork.com/ab/f/contracts/20874295)

**Comment by dmitry-fedyuk** [comment]
![01](https://user-images.githubusercontent.com/83204/47501602-babe0500-d887-11e8-9dc9-08dffcf26e5c.png)
